client/components/happychat/style.scss

```diff
@@ -284,10 +284,25 @@
 	padding: 0 8px;
 	margin: 6px 0;
 
+	a {
```

Comment:
This would be an exception since we don't necessarily control the markup that is rendered in the chats.